packages/ckeditor5-core/src/editor/editor.ts

```diff
@@ -300,6 +301,23 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
 		this.config = new Config<EditorConfig>( rest, defaultConfig );
 		this.config.define( 'plugins', availablePlugins );
 		this.config.define( this._context._getEditorConfig() );
+		this.config.define( 'sanitizeHtml', function( rawHtml ) {
+			/**
+			 * When using the HTML embed feature with the `htmlEmbed.showPreviews=true` option or the merge fields feature with
+			 * the `mergeFields.previewHtmlValues=true` option, it is strongly recommended to
+			 * define a sanitize function that will clean up the input HTML in order to avoid XSS vulnerability.
```

Comment:
```suggestion
			 * One of the editor features directly inserts unsanitized HTML code into the editor.
			 * It is strongly recommended to define a sanitize function that will clean up the input HTML in order to avoid XSS vulnerability.
```

Reply:
Changed so we don't need to update this message with every new feature that requires sanitization. Details are in the guide anyway.